### DISCUSSION TITLE
How to log account address after connecting in newer version

### DISCUSSION THREAD

**Discussion by HaniGannam** [WalletConnect/walletconnect-monorepo#4137] [Developer Support]
<script setup>
import { createWeb3Modal, defaultConfig, useWeb3Modal } from '@web3modal/ethers/vue'

// 1. Get projectId at https://cloud.walletconnect.com
const projectId = ''

// 2. Set chains
const mainnet = {
  chainId: 1,
  name: 'Ethereum',
  currency: 'ETH',
  explorerUrl: 'https://etherscan.io',
  rpcUrl: 'https://cloudflare-eth.com'
}

// 3. Create modal
const metadata = {
  name: 'My Website',
  description: 'My Website description',
  url: 'https://mywebsite.com',
  icons: ['https://avatars.mywebsite.com/']
}

const { open, close } = useWeb3Modal(createWeb3Modal({
  ethersConfig: defaultConfig({ metadata }),
  chains: [mainnet],
  projectId
}));

</script>

<template>
  <div>
    <w3m-button />
    <button @click="view">View</button>
  </div>
</template>

**Comment by glitch-txs** [comment]
Please provide a more descriptive explanation of your issue or what you are trying to accomplish, you can follow this guide as well: https://github.com/orgs/WalletConnect/discussions/3457
